core/api/workspace.josh.hcl

```diff
@@ -165,17 +171,28 @@ struct "log-description" {
 }
 
 struct "event" {
-	field "id" "string" {}
-	field "log" "string" {}
-	field "timestamp" "string" {}
-	field "message" "string" {}
+  field "id" "string" {}
+  field "log" "string" {}
+  field "timestamp" "string" {}
+  field "message" "string" {}
 }
 
 struct "process-description" {
-	field "id" "string" {}
+  field "id" "string" {}
   field "provider" "string" {}
-	field "name" "string" {}
+  field "name" "string" {}
+  field "running" "bool" {}
+}
+
+struct "component-status" {
+	field "component-id" "string" {}
 	field "running" "bool" {}
+  field "envVars" "map[string]string" {}
```

Comment:
i'm good with this file using spaces instead of tabs, not sure why it wasn't formatted correctly previously.

but this line and around this line looks messed up now. Is there some standard formatter we can configure somehow?

Reply:
Yes, I fixed the mixed spaces and tabs — and then reintroduced the problem. :facepalm: I did quickly look for a formatter but the only one I found hasn't been maintained in 6 years: https://github.com/fatih/hclfmt.

Will fix this again.